provider/offline.go

```diff
@@ -8,23 +8,25 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/mitchellh/mapstructure"
 	"sort"
 	"strings"
 	"time"
 
+	"github.com/mitchellh/mapstructure"
+
 	"github.com/featureform/metadata"
+	pc "github.com/featureform/provider/provider_config"
 	"github.com/google/uuid"
 )
 
 const (
 	MemoryOffline    Type = "MEMORY_OFFLINE"
-	PostgresOffline       = "POSTGRES_OFFLINE"
-	SnowflakeOffline      = "SNOWFLAKE_OFFLINE"
-	RedshiftOffline       = "REDSHIFT_OFFLINE"
-	SparkOffline          = "SPARK_OFFLINE"
-	BigQueryOffline       = "BIGQUERY_OFFLINE"
-	K8sOffline            = "K8S_OFFLINE"
+	PostgresOffline  Type = "POSTGRES_OFFLINE"
```

Comment:
these aren't required

Reply:
By these I mean adding the Type, it was fine before

Reply:
nbd if you want to keep it this way, just an FYI

Reply:
I looked into this and thought that it might just be an issue with my language server in VS Code, but after reading [this](https://stackoverflow.com/questions/64826695/set-type-for-entire-const-block) it appears if we want the alias to apply to all of the variants, we need to specify it on each line or use a single-line notation.

Reply:
Today I learned